stdlib/float64.gr

```diff
@@ -10,10 +14,45 @@ import {
   coerceFloat64ToNumber as toNumber
 } from "runtime/numbers"
 
+/**
+ * @section Conversions: Functions for converting between Numbers and the Float64 type.
+ */
+
+/**
+ * Converts a Number to a Float64.
+ *
+ * @param number: The value to convert
+ * @returns The Number represented as a Float64
+ * 
+ * @since v0.2.0
+ * @history v0.3.0: Function name changed from coerceNumberToFloat64 to fromNumber
```

Comment:
```suggestion
```
I checked https://github.com/grain-lang/grain/blob/v0.2.0/stdlib/float64.gr and it seems that these were actually renamed as the foreign import. Notice `as fromNumber`